awx/main/models/execution_environments.py

```diff
@@ -17,6 +17,7 @@ class Meta:
         ('always', _("Always pull container before running.")),
         ('missing', _("Only pull the image if not present before running.")),
         ('never', _("Never pull container before running.")),
+        ('newer', _("Pull if the image on the registry is newer than the one in the local containers storage.")),
```

Comment:
just to keep the description shorter, maybe this can be

`Only pull if a newer image is available in the registry.`

Reply:
Hey Seth! I just updated it, let me know if the PR looks good to you now!